src/react-apps/package.json

```diff
@@ -15,8 +15,8 @@
     "jest-environment-enzyme": "^7.0.1",
     "jest-enzyme": "^7.0.1",
     "lerna": "^3.4.3",
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
+    "react": "16.6.1",
+    "react-dom": "16.6.1",
```

Comment:
Why are these downgraded and removed the `^`?

Reply:
Hard-coded versions. Most of our apps are on 16.6.1. Of course we can upgrade all to 16.6.3?

Reply:
I see the point of locking the versions. It's just that the downgrade, if I'm not mistaken, it's a semantic release version, so when the last numbers goes up, that means that there are bug-fixes, and are compatible with all 16.6.x-versions. Don't see why we should downgrade bug-fixes.

Reply:
I have changed the version to 16.6.3